apps/examples/wifi_manager/wm_test/wm_test_connectbyrssi_test.c

```diff
@@ -268,8 +268,13 @@ static void wm_run_connect_by_rssi(void *arg)
 		memset(bssid, '\0', sizeof(bssid));
 		memset(connected_ap_mac, '\0', sizeof(connected_ap_mac));
 		_wt_scan(arg);		
-		_wt_connect(arg);		
+		_wt_connect(arg);
 				
+		if (bssid[0] == '\0' || connected_ap_mac[0] == '\0') {
```

Comment:
You can add these changes to your 1st commit since they seem connected to the same change.
You may not need a new commit just to add these.